widgets/progressbar/class.tx_mkforms_widgets_progressbar_Main.php

```diff
@@ -139,13 +133,8 @@ function initSteps() {
 	function getValue() {
 		$fValue = floatval(parent::getValue());
 
-		$fMin = $this->getMinValue();
 		$fMax = $this->getMaxValue();
 
-		if($fValue < $fMin) {
-			$mValue = $fMin;
```

Comment:
please dont remove this. This as to be fixed. $mValue should be changed to $fValue